### PR TITLE
iATS Payments: Pass email address outside of the billing address scope

### PR DESCRIPTION
**Pull request by naashton**
Give ability to pass customer details outside of the scope of the
billing address

CE-795

Unit: 18 tests, 178 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

Remote: 14 tests, 61 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

### CODE REVIEW

**Review by meagabeth — COMMENTED** [review]
Tests are passing! Does a unit test need to be written for this?

**Comment by naashton** [comment]
In this particular case, this is a specific user case where the customer wants to be able to pass customer data at the transaction level as opposed to passing it in the billing array. That being said, adding a unit test won't hurt.

File: test/remote/gateways/remote_iats_payments_test.rb [hunk 1]
```diff
@@ -66,7 +74,7 @@ def test_failed_refund
     purchase = @gateway.purchase(@amount, credit_card, @options)
     assert_success purchase
 
-    assert refund = @gateway.refund(@amount, purchase.authorization)
```
Comment (therufs) [inline test/remote/gateways/remote_iats_payments_test.rb:69]: weird

**Review by therufs — APPROVED** [review]
lgtm; all local and remote tests pass!

File: test/remote/gateways/remote_iats_payments_test.rb (hunk 1 above)
Comment (naashton) [reply test/remote/gateways/remote_iats_payments_test.rb:69]: Super weird

**Pull request merged**